postgres_mixin/alerts/postgres.libsonnet

```diff
@@ -219,7 +219,12 @@
               description: '{{ $labels.instance }} replication lag in bytes is too large, which might indicate replication issues or network bottlenecks.',
               summary: 'PostgreSQL replication lag in bytes too large.',
             },
-            expr: '(pg_xlog_position_bytes{asserts_env!=""} and pg_replication_is_replica{asserts_env!=""} == 0) - on (job, service, asserts_env, asserts_site) group_right(instance) (pg_xlog_position_bytes{asserts_env!=""} and pg_replication_is_replica{asserts_env!=""} == 1) > 1e+09',
+            expr: |||
+              (pg_xlog_position_bytes{} and pg_replication_is_replica{} == 0)
+              - on (job, service) group_right(instance) (
+                pg_xlog_position_bytes{} and pg_replication_is_replica{} == 1
+              ) > 1e+09
```

Comment:
Same thing here, specifically `1e+09`. Needs to be explained

Reply:
This alert has been removed for now - see comment above